apps/frontend/src/components/global/upload_tabs/GroupFileList.vue

```diff
@@ -0,0 +1,228 @@
+<template>
+  <div>
+    <v-text-field
+      v-model="search"
+      class="px-3 pb-1"
+      append-icon="mdi-magnify"
+      label="Search"
+      hide-details
+    />
+    <DeleteDialog
+      v-model="deleteItemDialog"
+      type="file"
+      @cancel="deleteItemDialog = false"
+      @confirm="deleteItemConfirm"
+    />
+    <v-data-table
+      v-model="selectedFiles"
+      :headers="headers"
+      :loading="loading"
+      :items="filteredFiles"
+      group-by="group"
+      show-select
+      dense
```

Comment:
Groups are currently closable which is a little weird, is that possible to disable?

![Screen Shot 2021-06-09 at 9 50 37 AM](https://user-images.githubusercontent.com/2308869/121367274-2d306600-c908-11eb-8097-dfcd9c1190e8.png)